### PR TITLE
Arp partitioning

### PR DESCRIPTION
**Pull request by fhajredini**
meta-arp-extras: new fstab for ARP 
conf/variant: include meta-arp-extras

### CODE REVIEW

File: conf/variant/arp/bblayers.conf.sample [hunk 1]
```diff
@@ -8,4 +8,5 @@ BBLAYERS += "\
     ${BSPDIR}/sources/meta-pelux/meta-intel-extras  \
```
Comment (sashko) [inline conf/variant/arp/bblayers.conf.sample]: could you please sort them out somehow in a logical way?

File: meta-arp-extras/conf/layer.conf [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+# We have a recipes directory, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb ${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "pelux-bsp-intel-layer"
+BBFILE_PATTERN_pelux-bsp-intel-layer := "^${LAYERDIR}/"
+
+BBFILE_PRIORITY_pelux-bsp-intel-layer = "10"
+
+LAYERSERIES_COMPAT_pelux-bsp-intel-layer = "sumo thud warrior"
```
Comment (sashko) [inline meta-arp-extras/conf/layer.conf]: arp

File: meta-arp-extras/recipes-core/base-files/base-files/fstab [hunk 3]
```diff
@@ -0,0 +1,5 @@
+/dev/root            /                    auto       defaults              1  1
```
Comment (sashko) [inline meta-arp-extras/recipes-core/base-files/base-files/fstab]: fix indentations please

File: meta-arp-extras/recipes-core/base-files/base-files_%.bbappend [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#
+#   Copyright (C) 2017 Pelagicore AB
```
Comment (sashko) [inline meta-arp-extras/recipes-core/base-files/base-files_%.bbappend]: 2019

File: meta-arp-extras/conf/layer.conf [hunk 5]
```diff
@@ -0,0 +1,12 @@
+BBPATH .= ":${LAYERDIR}"
+
+# We have a recipes directory, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb ${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "pelux-bsp-intel-layer"
+BBFILE_PATTERN_pelux-bsp-intel-layer := "^${LAYERDIR}/"
+
+BBFILE_PRIORITY_pelux-bsp-intel-layer = "10"
```
Comment (sashko) [inline meta-arp-extras/conf/layer.conf]: arp

File: meta-arp-extras/conf/layer.conf [hunk 6]
```diff
@@ -0,0 +1,12 @@
+BBPATH .= ":${LAYERDIR}"
+
+# We have a recipes directory, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb ${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "pelux-bsp-intel-layer"
```
Comment (sashko) [inline meta-arp-extras/conf/layer.conf]: arp

**Comment by martin-ejdestig** [comment]
> ARP is supposed to use fstab from intel, but instead it was using the default one from poky. This commit contains the fstab that will be used on ARP.

Can this not be controlled by layer priorities? Does `bitbake -e <whatever it is that provides the fstab>` tell you why it does not? Compare what the difference is between an Intel build and an ARP one.

File: conf/variant/arp/bblayers.conf.sample (hunk 1 above)
Comment (fhajredini) [reply conf/variant/arp/bblayers.conf.sample]: Done!

**Comment by fhajredini** [comment]
> > ARP is supposed to use fstab from intel, but instead it was using the default one from poky. This commit contains the fstab that will be used on ARP.
> 
> Can this not be controlled by layer priorities? Does `bitbake -e <whatever it is that provides the fstab>` tell you why it does not? Compare what the difference is between an Intel build and an ARP one.

I went through the log and didnt notice a difference. I will go ahead and merge this as for now, this seems the only way to have this fixed.

**Pull request merged**